features/frontend/checkout_payment.feature

```diff
@@ -22,10 +22,10 @@ Feature: Checkout Payment
             | zone | name        |
             | UK   | DHL Express |
           And the following payment methods exist:
-            | name        | gateway    | enabled |
-            | Credit Card | stripe     | yes     |
-            | PayPal      | paypal     | yes     |
-            | PayPal PRO  | paypal_pro | no      |
+            | name        | gateway    | enabled | calculator | calculator_configuration |
+            | Credit Card | stripe     | yes     | fixed      | amount: 10               |
+            | PayPal      | paypal     | yes     | fixed      | amount: 50               |
+            | PayPal PRO  | paypal_pro | no      | percent    | amount: 10               |
```

Comment:
Unfortunately, ``getConfiguration`` method in ``DefaultContext`` cast 'amount' value to integer.
So for now i will leave this as it is and maybe change in another PR ;)